testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/MultinodeKafka.java

```diff
@@ -20,27 +20,28 @@
 import io.trino.tests.product.launcher.env.Environment;
```

Comment:
cmt msg should mention the bug that is fixed. Multinode is only used here for tests coverage.